src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -918,14 +906,6 @@ FixedwingPositionControl::control_auto(const hrt_abstime &now, const Vector2d &c
 		publishOrbitStatus(current_sp);
 	}
 
-	// update lateral guidance timesteps for slewrates
-	if (_param_fw_use_npfg.get()) {
-		_npfg.setDt(dt);
```

Comment:
Where did this go?

Reply:
here:

https://github.com/PX4/PX4-Autopilot/blob/3038b5cf50cd0bedb6bd49fcae2b49fddba8d980/src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp#L2385-L2391